ardour5.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
```

Comment:
Why is it here, if you later use the `desktop-file-edit` command?

Reply:
no idea why I added it to git. Will remove

Reply:
removed.